rootfs-builder/euleros/rootfs_lib.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+#
+# Copyright (c) 2017 Intel Corporation
```

Comment:
This copyright should probably be Huawei, or Intel+Huawei?

/cc @grahamwhaley

Reply:
Looks like it is a modified copy of the original at (this or similar) https://github.com/kata-containers/osbuilder/blob/master/rootfs-builder/centos/rootfs_lib.sh - in which case, yes, probably makes sense to add a Huawei notice in there, just to clarify its history and to make it clear it is not purely Intel code any more.
(IANAL :-))

Reply:
Yes, please modify it to Huawei.also s/2017/2018/, thanks @liangchenye:smile:

Reply:
OK, I'll change it to 2018 Huawei ..